packages/harmony/src/components/button/Button/types.ts

```diff
@@ -7,7 +7,7 @@ export type HTMLButtonProps = ComponentPropsWithoutRef<'button'>
 
 export type ButtonVariant = 'primary' | 'secondary' | 'tertiary' | 'destructive'
 
-export type ButtonSize = 'small' | 'default' | 'large'
+export type ButtonSize = 'xs' | 'small' | 'default' | 'large'
```

Comment:
feels a bit off to have acronym next to spelled out ones. probably more trouble than its worth to change though

Reply:
totally agree, im actually leaning towards using s | m | l so its similar to our spacing values, what you think?

Reply:
yea i agree, i like the acronyms